nutri-track-bot/src/main/java/com/nutritrack/bot/service/telegram/factory/TelegramFactory.java

```diff
@@ -27,13 +29,10 @@ public static TelegramWebhookBot webhookBot(String token,
                                                 String path,
                                                 Function<Update, BotApiMethod> onUpdate) {
         return new TelegramWebhookBot(token) {
-
-            {
-                registerCommands();
```

Comment:
Without this method, the bot will start without any commands, and if a user tries to use the 'START' command, it will give them nothing. 

Please leave it as it is.